iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerLexicalConstant.java

```diff
@@ -96,6 +96,8 @@ public class DockerLexicalConstant {
 
   public static final String IMAGE_ALIAS = "[-a-zA-Z0-9_\\.]+";
 
+  public static final String GARBAGE = "[^\n\r]+";
```

Comment:
That's a funny name :D But I have nothing better to suggest (`IGNORED`? `IGNORED_TRAILING`?)

Reply:
Haha same I couldn't come with something better :sweat_smile: I will try to replace it with something more meaningful.